packages/unplugin-vue-i18n/src/core/resource.ts

```diff
@@ -587,40 +555,21 @@ function isCustomBlock(query: VueQuery): boolean {
 
 function getOptions(
   filename: string,
-  isProduction: boolean,
+  prod: boolean,
   query: VueQuery,
-  sourceMap: boolean,
-  {
-    jit,
-    transformI18nBlock,
-    ...opts
-  }: Pick<
-    ResolvedOptions,
-    | 'globalSFCScope'
-    | 'strictMessage'
-    | 'allowDynamic'
-    | 'escapeHtml'
-    | 'onlyLocales'
-    | 'forceStringify'
-    | 'transformI18nBlock'
-  > & { inSourceMap: RawSourceMap | undefined; jit: boolean }
-): Record<string, unknown> {
-  const mode: DevEnv = isProduction ? 'production' : 'development'
-
-  const baseOptions = {
+  opts: CodeGenOptions & Pick<ResolvedOptions, 'globalSFCScope'>
+): CodeGenOptions {
+  const mode: DevEnv = prod ? 'production' : 'development'
+
+  const baseOptions: CodeGenOptions = {
     ...opts,
     filename,
-    sourceMap,
-    jit,
+    jit: true,
```

Comment:
@kazupon 
While refactoring it seemed like all option passing eventually resulted in `jit: true` being passed, is this correct?